tests/bgp/test_bgp_sentinel.py

```diff
@@ -394,9 +394,15 @@ def prepare_bgp_sentinel_routes(rand_selected_dut, common_setup_teardown, bgp_co
             announce_route(ptfip, lo_ipv6_addr, route, ptf_bp_v6, BGP_SENTINEL_PORT_V6, community)
 
     time.sleep(10)
+
```

Comment:
Prefer to check route received from neighbors here.
# Check if DUT receives the routes that announced from ptf
  for ibgp_session in ibgp_sessions:
      if request.param == "IPv4":
          cmd = "vtysh -c \'show bgp ipv4 neighbors {} received-routes json\'".format(ptf_bp_v4)
      else:
          cmd = "vtysh -c \'show bgp ipv6 neighbors {} received-routes json\'".format(ptf_bp_v6)
      output = json.loads(duthost.shell(cmd)['stdout'])
      logger.debug(output)